### ISSUE TITLE
Is there a cheat sheet for the keybindings and what can be done when debugging?

### ISSUE THREAD

**Issue by dfinke** [PowerShell/vscode-powershell#85]

**Comment by daviwil** [comment]
Nah, not yet. I need to start putting together a user guide soon with this sort of information, just haven't had the time.

**Comment by rkeithhill** [comment]
And are we going to change the binding for opening ISE from `Ctrl+Alt+i` to `Ctrl+Shift+i`?

**Comment by daviwil** [comment]
Probably good to decide on that before shipping this release so we don't confuse people.  I don't know exactly why they recommend against `Ctrl+Alt` but there's a definitely conflict with `Ctrl+Shift+i` in loading the Chrome Dev Tools.  Not the worst thing, but could be annoying in developer mode.

**Comment by rkeithhill** [comment]
What is the `when` condition for the Chrome Dev Tools?  Is it just `editorTextFocus`?  Or is it also limited by `editorLangId == 'html'`?  If so, then we can use that shortcut and it won't interfere because we would only take it for `editorLangId = 'powershell'`.

**Comment by daviwil** [comment]
I think it's a global shortcut.  I just tried changing the shortcut to `Ctrl+Shift+i` and the hilarious thing is that it launches both the ISE and the Chrome Dev Tools when you're in extension development mode.  Only if your cursor is in a PowerShell file though.  As far as I'm concerned, that's OK.  I don't seem to get a complaint from VS Code about the mapping either.  I'll go ahead and change the shortcut now.

**Comment by TylerLeonhardt** [comment]
Some of this is covered in https://docs.microsoft.com/en-us/powershell/scripting/components/vscode/how-to-replicate-the-ise-experience-in-vscode?view=powershell-7

and a tad in https://docs.microsoft.com/en-us/powershell/scripting/components/vscode/using-vscode?view=powershell-7

but it would be nice to have one place
